Bottom: a4d711395430c2585b9698c807d704f5f3071c6d
Top:    a820607d2ceadb148d2eb04c532fc14de4e8383a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 20:18:56 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
